src/app.js

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2015 Colin Clark
+
+Licensed under the 3-Clause "New" BSD license.
+You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the 3-Clause "New" BSD License at
+https://github.com/colinbdclark/infusion-electron/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion"),
+    electron = fluid.registerNamespace("electron");
+
+// Monkey-patch fluid.log for Electron's broken console.log.
+fluid.doLog = function (args) {
+    if (typeof (console) !== "undefined") {
+        if (console.debug) {
+            console.debug(args.join(" "));
+        } else if (typeof (console.log) === "function") {
+            console.log(args.join(" "));
+        }
+    }
+};
+
+
+fluid.defaults("electron.app", {
+    gradeNames: "fluid.modelComponent",
+
+    members: {
+        app: require("app")
```

Comment:
This is undesirable - I think there's a lot of value in placing all requires at the head of a file, and then assigning them to a global namespace if more generally useful. In fact I'm not clear exactly what is resolved by "app" here - I don't see an entry in package.json and I don't believe that this file can resolve to itself, and in any case it doesn't export anything ....